api/src/pcapi/core/bookings/models.py

```diff
@@ -63,6 +63,7 @@ class BookingCancellationReasons(enum.Enum):
     BACKOFFICE_BENEFICIARY_REQUEST = "BACKOFFICE_BENEFICIARY_REQUEST"
     BACKOFFICE_OFFER_MODIFIED = "BACKOFFICE_OFFER_MODIFIED"
     BACKOFFICE_OFFER_WITH_WRONG_INFORMATION = "BACKOFFICE_OFFER_WITH_WRONG_INFORMATION"
+    BACKOFFICE_OFFERER_END_OF_ACTIVITY = "BACKOFFICE_OFFERER_END_OF_ACTIVITY"
```

Comment:
Goût personnel : je trouve « end of activity » pas si évident ; quelque chose du genre « business closed », donc `BACKOFFICE_OFFERER_BUSINESS_CLOSED`, par exemple, ou une idée claire avec « closed », me semblerait plus intuitive.